moveit_py/src/moveit/moveit_ros/moveit_cpp/planning_component.cpp

```diff
@@ -73,7 +74,7 @@ plan(std::shared_ptr<moveit_cpp::PlanningComponent>& planning_component,
     if (solution_selection_function && stopping_criterion_callback)
     {
       return planning_component->plan(*const_multi_plan_parameters, std::ref(*solution_selection_function),
-                                      *stopping_criterion_callback);
+                                      *stopping_criterion_callback, planning_scene);
     }
     else if (solution_selection_function)
     {
```

Comment:
Line 81 is missing the planning_scene argument.

Reply:
https://github.com/ros-planning/moveit2/blob/cc7a0d422b294ffc7c01a52df074140bb3c5aa15/moveit_ros/planning/moveit_cpp/src/planning_component.cpp#L211-L212 Does not allow for a planning scene argument.
I can see if i can add that to the function. Or i can throw an exception if there is this combinaiton was added?